com.unity.render-pipelines.high-definition/CHANGELOG.md

```diff
@@ -112,8 +112,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 - Changed Window/Render Pipeline/HD Render Pipeline Wizard to Window/Rendering/HDRP Wizard
 - Removed the material pass probe volumes evaluation mode.
-- Changed GameObject/Rendering/Density Volume to GameObject/Volume/Density Volume
-- Changed GameObject/Rendering/Decal Projector to GameObject/Decal Projector
```

Comment:
why this line have disappeard? Changed GameObject/Rendering/Decal Projector to GameObject/Decal Projector

Reply:
TA decided to go again on Rendering

https://unity.slack.com/archives/CH5875RNF/p1615914336010700